homeassistant/components/command_line/binary_sensor.py

```diff
@@ -115,28 +103,23 @@ def __init__(
         unique_id: str | None,
     ) -> None:
         """Initialize the Command line binary sensor."""
-        TemplateEntity.__init__(
-            self,
-            hass,
-            config=config,
-            fallback_name=name,
-            unique_id=unique_id,
-        )
         self.data = data
+        self._attr_name = name
         self._attr_device_class = device_class
         self._attr_is_on = None
         self._payload_on = payload_on
         self._payload_off = payload_off
         self._value_template = value_template
+        self._attr_unique_id = unique_id
 
     async def async_update(self) -> None:
         """Get the latest data and updates the state."""
         await self.hass.async_add_executor_job(self.data.update)
         value = self.data.value
 
         if self._value_template is not None:
-            value = await self.hass.async_add_executor_job(
-                self._value_template.render_with_possible_json_value, value, False
+            value = self._value_template.async_render_with_possible_json_value(
+                value, None
```

Comment:
This changes the error value to `None`, should the binary sensor's state also be `None` when this happens?

Reply:
I have cleaned this as well so it works same between the platforms resulting in `None`.